app/shared_features/set_card/SetRestRow.js

```diff
@@ -11,14 +11,17 @@ class SetRestRow extends PureComponent {
 
     render() {
         if (this.props.item.isCollapsed) {
-            var margin = 0;
+            var marginTop = 0;
+            var marginBottom = 15;
```

Comment:
- [ ] marginBottom needs to be based on if there's a DELETE button as well. If delete button, less margin. IF not delete button, more margin.

Reply:
A delete button only shows when the item is expanded...